### PR TITLE
Added links to Column named Filename in Release  #11702

### PR DESCRIPTION
**Pull request by Lakshya1605**
#11702 @asmeurer  Changed as requested please verify .Thank you

### CODE REVIEW

File: release/fabfile.py [hunk 1]
```diff
@@ -552,6 +554,11 @@ def tag(name):
         table.append("<%s>" % name)
         yield
         table.append("</%s>" % name)
+    @contextmanger
```
Comment (asmeurer) [inline release/fabfile.py]: You spelled this wrong.

File: release/fabfile.py [hunk 2]
```diff
@@ -552,6 +554,11 @@ def tag(name):
...
+    def a_href(link):
+        table.append("<a href=\""+"%s" % link + "\">")
```
Comment (asmeurer) [inline release/fabfile.py]: You can string format the whole string, like `'<a href="%s">' % link`.

**Review by asmeurer — CHANGES_REQUESTED** [review]

File: release/fabfile.py [hunk 3]
```diff
@@ -563,8 +570,9 @@ def tag(name):
             name = get_tarball_name(key)
             with tag('tr'):
                 with tag('td'):
-                    with tag('b'):
-                        table.append(name)
+                    with a_href('https://github.com/sympy/sympy/releases/download/sympy-'+'%s' %version+'/'+'%s' %name):
```
Comment (asmeurer) [inline release/fabfile.py]: Also here you can just use one string. `'https://github.com/sympy/sympy/releases/download/sympy-%s/%s' % (version, name)`.

**Comment by moorepants** [comment]
@Lakshya1605 can you verify that this works.

**Comment by Abdullahjavednesar** [comment]
@asmeurer Are the changes good? Is it working?

**Pull request merged**